docs/data/pages.ts

```diff
@@ -70,6 +70,7 @@ const pages: MuiPage[] = [
         ],
       },
       { pathname: '/x/react-data-grid/export' },
+      { pathname: '/x/react-data-grid/clipboard' },
```

Comment:
I think the title "Clipboard" is obvious to us, and perhaps to users that are already familiar with the export page, but I was wondering if it would be helpful to have the title on the navigation menu be more clear on what we mean with this feature.

<img width="297" alt="Screenshot 2023-04-12 at 11 19 04" src="https://user-images.githubusercontent.com/550141/231413581-9376b385-79e6-4e55-ab12-21df168f5ed2.png">

Reply:
Agree, "Copy and paste" seems more clear.
Do you think it's OK to keep `clipboard` in the URL?